prisma/seed/index.ts

```diff
@@ -225,6 +233,32 @@ const users = async () => {
     })),
   })
 
+  const organizationVersionsCUT = await prisma.organizationVersion.createManyAndReturn({
+    data: organizations.map((organization, index) => ({
+      organizationId: organization.id,
+      isCR: index % 2 === 0,
```

Comment:
```suggestion
      isCR: false,
```

Tous les utilisateurs CUT sont non CR